api/openapi/specs/common/resource.yaml

```diff
@@ -123,7 +125,7 @@ components:
         conf:
           description: The actual conf generated
           type: object
-          x-go-type: 'interface{}'
```

Comment:
Can I remove these? If so should I also remove the `x-go-type: 'map[string]string'` form the `tags` property?

Reply:
Looks like go needs these, resolving.